Tools/TestWebKitAPI/Tests/WebKitCocoa/UIDelegate.mm

```diff
@@ -1039,6 +1039,39 @@ - (void)_webView:(WKWebView *)webview mouseDidMoveOverElement:(_WKHitTestResult
 
 static BlockPtr<NSEvent*(NSEvent*)> gEventMonitorHandler;
 
+@interface TestLocalEventObserver : NSObject {
+    NSEventMask _mask;
+    id _block;
+    BOOL _isAdditive;
+}
++ (void)initialize;
+- (id)initMatchingEvents:(NSEventMask)mask handler:(NSEvent* (^)(NSEvent*))block;
+- (void)invalidate;
+- (void)dealloc;
+- (void)recomputeObserverMask;
+@end
+
+@implementation TestLocalEventObserver
++ (void)initialize {
+}
+
+- (id)initMatchingEvents:(NSEventMask)mask handler:(NSEvent* (^)(NSEvent*))block {
+    self = [super init];
+    return self;
+}
+
+- (void)dealloc {
```

Comment:
these braces should be on a newline right?

Reply:
Correct, fixed!